integration_tests/apps/test-1.0/app-canary.yaml

```diff
@@ -0,0 +1,4 @@
+runtime: aspnetcore
+env: flex
+beta_settings:
+  image: nextcanary
```

Comment:
is this the only required configuration for running these on the nextcanary image? could you link some documentation here (just for my own sake)?